.github/workflows/release.yaml

```diff
@@ -16,11 +16,36 @@ jobs:
         with:
           fetch-depth: 0
 
-      - name: Tag
-        uses: tvdias/github-tagger@v0.0.2
+      - name: Setup Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: "1.14.5"
+
+      - name: Cache
+        uses: actions/cache@v2
         with:
-          repo-token: "${{ secrets.CI_TOKEN }}"
-          tag: "${{ github.event.inputs.version }}"
+          path: ~/go/pkg/mod
+          key: ${{ runner.os }}-go-${{ hashFiles('**/go.sum') }}
+          restore-keys: |
+            ${{ runner.os }}-go-
+
+      - name: Install tools
+        run: ./setup.sh
+
+      - name: Tag
+        run: |
+          git config user.name "$GITHUB_ACTOR"
+          git config user.email "$GITHUB_ACTOR@users.noreply.github.com"
+
+          tag='${{ github.event.inputs.version }}'
+          git tag --annotate --message "Tag for release $tag" "$tag"
+          git push "https://x-access-token:${{ secrets.GITHUB_TOKEN }}@github.com/$GITHUB_REPOSITORY" "refs/tags/$tag"
+
+      - name: Check License Headers
+        run: mage -v checkLicenseHeaders
+
+      - name: Lint
```

Comment:
should maybe the checks happen before the tag?

Reply:
Ouch, good catch

Reply:
Fixed